doc/source/manual/xtrabackup_sst.rst

```diff
@@ -35,6 +35,13 @@ under ``[sst]``.
      on either donor or joiner node, not during ``mysqld`` startup.
 
    * SST options must be specified in the main :file:`my.cnf` file.
+   
+The following options can be added to the [sst] section of the configuration file to control SST timeouts:
+
+* ``sockopt``
+* ``donor-timeout`` - the value of the `connect-timeout` on the donor node. The default valueis 10 seconds.
+* ``joiner-timeout`` - The time the joiner waits for the SST transfer to start. The default value is 60 seconds.
+* ``sst-idle-timeout``
```

Comment:
If the SST process stuck for this amount of time (no data transferred), it will be interrupted. Now it works on both sides: joiner and donor. The section below, describing details should be updated as well.